style.css

```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
 ul, li{
     list-style: none;
 }
+
 .logo_header{
-    margin-top: 30px;
+    padding-top: 41px;
     margin-left: 184px;
+    padding-right: 123px;
 }
-.wrapper{
+.header__inner{
     display: flex;
+    justify-content: space-around;
+}
+.nav__link{
+    padding-top: 44px;
+    margin-left: 46px;
+    text-decoration: none;
+    color: #696871;
+    font-family:'DM Sans', sans-serif;
+    font-size: 0.9375rem;
+    line-height: 19px;
+}
+.nav__link:hover{
+    color:#19191B ;
+    text-decoration:underline;
+}
+.nav{
+    padding-top: 44px;
+}
+.btn__header{
+    padding-top:30px ;
+    margin-left: 159px;
+    margin-right: 180px;
+}
+.btn__inner{
+    font-family:'DM Sans', sans-serif;
+    font-size: 17px;
+    line-height: 22px;
+    text-align: center;
+    background-color: #FF7143;
+    color:white ;
```

Comment:
you should write color in one style
if write color as HEX, write it

Reply:
done